curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/understand-string-immutability.md

```diff
@@ -11,14 +11,14 @@ dashedName: understand-string-immutability
 
 In JavaScript, `String` values are <dfn>immutable</dfn>, which means that they cannot be altered once created.
 
-For example, the following code:
+For example, the following code cannot change the first letter of the string `Bob` to a `J`:
```

Comment:
perhaps
```suggestion
For example, the following code will produce an error because the letter `B` in the string `Bob` cannot be changed to the letter `J`:
```